README.md

```diff
@@ -10,13 +10,15 @@ This is the rendering application for the Book a secure move service.
 Book a secure move is part of the HMPPS Prisoner Escort and Custody
 Service (PECS) programme.
 
-## Dependencies
+## Quick Start
+
+### Major Dependencies
 
 - [Node.js](https://nodejs.org/en/) (>= 14)
 - [NPM](https://www.npmjs.com/) (>= 6.14)
 - [Redis](https://redis.io/) (>= 5.0.5)
```

Comment:
For running the app in production, yes, but not in dev.

Reply:
@solidgoldpig Oops - what have I missed?

Reply:
Er, you don't need to use redis. The e2e tests. for instance, don't. And whether you use it while developing is a matter of choice. Personally I prefer not to have to login again just because the app has restarted, but sometimes it can be a help/

Reply:
@solidgoldpig Hmmm... I remember I had issues running the app in `develop` mode without redis - they went away when I accidentally installed Redis globally. 

What configuration do you use to run without it?

Reply:
@solidgoldpig Ah, so when running without redis an `in-memory` session is used, which is lost upon a server restart. Sometimes useful, sometimes not - I'll update the docs accordingly.